critiquebrainz/db/review.py

```diff
@@ -40,6 +40,20 @@ def to_dict(review, confidential=False):
     return review
 
 
+def get_top_languages():
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT  language,
+                    COUNT(id) AS count
+            FROM review
+            GROUP BY language
+            ORDER BY count
+            DESC LIMIT 10
+            """))
+        languages = result.fetchall()
```

Comment:
This fetches the tuple of (`language`, `count`). You need to get the first entry of the tuples in the list `language` (the code of the language).

Reply:
Do you have the CritiqueBrainz environment setup locally? Did you try running your changes locally? :)

Reply:
This needs to be resolved. I get the error:
```
critiquebrainz_1        | Traceback (most recent call last):
critiquebrainz_1        |   File "manage.py", line 14, in <module>
critiquebrainz_1        |     application = DispatcherMiddleware(frontend.create_app(), {
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/__init__.py", line 109, in create_app
critiquebrainz_1        |     from critiquebrainz.frontend.views.review import review_bp
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/views/review.py", line 14, in <module>
critiquebrainz_1        |     from critiquebrainz.frontend.forms.review import ReviewCreateForm, ReviewEditForm, ReviewReportForm
critiquebrainz_1        |   File "/code/critiquebrainz/frontend/forms/review.py", line 27, in <module>
critiquebrainz_1        |     other_languages_codes.remove(language)
critiquebrainz_1        | ValueError: list.remove(x): x not in list
```
After this line of code, you can fix this error by adding this line:
`languages = [language[0] for language in languages]`